File: CompiladoresProyFinal/app_generada.js
const express = require('express');
const app = express();
app.use(express.json());

app.get('/api/listar', (req, res) => {
    res.status(200).json({ message: 'Endpoint /api/listar ejecutado con éxito' });
});

app.post('/api/crear', (req, res) => {
    res.status(200).json({ message: 'Endpoint /api/crear ejecutado con éxito' });
});

app.get('/api/buscar/:id', (req, res) => {
    res.status(200).json({ message: 'Endpoint /api/buscar/:id ejecutado con éxito' });
});

app.listen(3000, () => {
    console.log('Servidor MiApiIncreible corriendo en http://localhost:3000');
});

File: CompiladoresProyFinal/app_generada.py
from flask import Flask, jsonify, request
app = Flask('UserManagement')

@app.route('/api/v1/users', methods=['GET'])
def get__users():
    return jsonify({'message': 'Endpoint /api/v1/users ejecutado con exito'}), 200

@app.route('/api/v1/users', methods=['POST'])
def post__users():
    return jsonify({'message': 'Endpoint /api/v1/users ejecutado con exito'}), 200

@app.route('/api/v1/users/<id>', methods=['GET'])
def get__users_id():
    return jsonify({'message': 'Endpoint /api/v1/users/<id> ejecutado con exito'}), 200

@app.route('/api/v1/users/<id>', methods=['PUT'])
def put__users_id():
    return jsonify({'message': 'Endpoint /api/v1/users/<id> ejecutado con exito'}), 200

@app.route('/api/v1/users/<id>', methods=['DELETE'])
def delete__users_id():
    return jsonify({'message': 'Endpoint /api/v1/users/<id> ejecutado con exito'}), 200

if __name__ == '__main__':
    app.run(debug=True, port=5000)